### ISSUE TITLE
Styling FormLabel without wrapping it with a FormControl.

### ISSUE THREAD

**Issue by luskafaria** [chakra-ui/chakra-ui#3929]
# 🐛 Bug report

Hi, I'm trying to style a FormLabel component but the styles don't get applied unless I wrap it with a FormControl is that on purpose or a bug?

## 💥 Steps to reproduce
 
Here's a sandbox: https://codesandbox.io/s/clever-easley-n18zn?file=/src/App.js:0-667

## 🧐 Expected behavior

I expected that I could style the FormLabel without wrapping it with a FormControl

## 🌍 System information

"@chakra-ui/react": "1.6.0",
"@emotion/react": "11.1.5",
"emotion/styled": "11.3.0",
"framer-motion": "4.1.11",
"react": "17.0.2",

| Software         | Version(s) |
| ---------------- | ---------- |
| Chakra UI        |      1.6.0      |
| Browser          |      Chrome - latest      |
| Operating System |      Ubuntu - latest      |

## 📔 Notes:

I'm upgrading my Chakra version from 0.8 to 1.6. With the 0.8 version was possible to style the FormLabel without wrapping it with a Form Control, that's why I created this issue.

PS: My English is kinda rusty, so let me know if something in my text sounds weird.

Thanks for this amazing lib!

**Comment by segunadebayo** [comment]
Good catch @luskafaria!

I just pushed a fix for this. We'll release an update shortly.

If the issue persists after upgrading, I'll re-open it.

**Comment by anton-johansson** [comment]
For anyone else coming here, this is the commit that fixed it:
https://github.com/chakra-ui/chakra-ui/commit/620f0b7d756ffb6bfc6ddf0459e96f774ffbb9be

Released in `@chakra-ui/react@1.6.2`.

**Comment by oyatek** [comment]
I still can't understand how can I style FromControl.
This is what I've tried and it doesn't work:
```
  const theme = extendTheme({
    FormLabel: {
      fontWeight: 'bold',
      fontSize: 'sm'
    }
  })
```
